webapp/app/js/perf.js

```diff
@@ -20,7 +20,7 @@ perf.provider('thServiceDomain', function() {
 perf.value('seriesColors', [ 'red', 'green', 'blue', 'orange', 'purple' ]);
 
 perf.factory('seriesSummary', ['seriesColors', function(seriesColors) {
-  return function(signature, signatureProps, projectName, optionCollectionMap,
+  return function(signature, signatureProps, projectName, visibility, optionCollectionMap,
```

Comment:
I believe you need to change the way this is called from the series selector UI. With this enabled, I see a lot of "undefined" when trying to select a series in the UI.